pyproject.toml

```diff
@@ -10,4 +10,102 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "cheroot"
+description = "Highly-optimized, pure-python HTTP server"
+authors = [
+    { name = "CherryPy Team", email = "team@cherrypy.dev" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Framework :: CherryPy",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: Implementation",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: Jython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+    "Topic :: Internet :: WWW/HTTP :: WSGI",
+    "Topic :: Internet :: WWW/HTTP :: WSGI :: Server",
+    "Typing :: Typed",
+]
+keywords = [
+    "http",
+    "server",
+    "ssl",
+    "wsgi",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "importlib_metadata; python_version < '3.8'",
```

Comment:
This doesn't make sense anymore, provided what's written two lines above. Looks like it was forgotten when Python 3.6 was being dropped...
```suggestion
```